packages/models/tests/server/organizations_test.coffee

```diff
@@ -32,3 +33,73 @@ describe 'Organization', ->
       expect(organization.validate("name")).not.to.be.ok
       organization.set('name', 'Test')
       expect(organization.validate("name")).to.be.ok
+
+  it 'can have members', (test, waitFor) ->
+    memberProfile = new UserProfile()
+    memberProfile.set(fullName: 'Snoopy')
+    id = memberProfile.save()
+    organization.save()
+    organization.addMember(id)
+    expect(
+      organization.getMemberProfiles().map((x)-> x.fullName)
+    ).to.include('Snoopy')
+
+  it 'can have admins', (test, waitFor) ->
+    memberProfile = new UserProfile()
+    memberProfile.set(fullName: 'TestUser')
+    memberProfile.save()
+    organization.set('name', 'TestOrg')
+    organization.save()
+    organization.addAdmin(memberProfile._id)
+    expect(
+      organization.getAdminProfiles().map((x)-> x.fullName)
+    ).to.include('TestUser')
```

Comment:
I guess we should decide what happens if you add an admin who's not a member. Should it throw an error, should we make that person a member as well? And whatever we do we should have a test for.